sass/README.md

```diff
@@ -16,3 +16,61 @@ Notable exceptions:
 * prefer double quotes
 
 [1]: http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/
+
+## whitespace
+
+Use space between neighboring nested blocks, but not before or after.
+
+### Good
+
+```scss
+.btn {
+  font-size: 15px;
```

Comment:
💄 should probably sort the rules in all examples since that's what we do in real SCSS.